internal/csi-addons/rbd/replication_test.go

```diff
@@ -438,58 +439,114 @@ func TestCheckRemoteSiteStatus(t *testing.T) {
-			nil,
-			"",
+			name: "description without last_snapshot_bytes",
+			//nolint:lll // sample output cannot be split into multiple lines.
+			description: `replaying, {"bytes_per_second":0.0,"last_snapshot_sync_seconds":56743,"local_snapshot_timestamp":1684675261,"remote_snapshot_timestamp":1684675261,"replay_state":"idle"}`,
+			info: localstruct{
+				lastsyncduration: durationpb.New(duration),
+				lastsynctime:     timestamppb.New(time.Unix(1684675261, 0)),
+				lastsyncbytes:    0,
+			},
+			expectedErr: "",
 		},
 		{
-			"description with invalid JSON",
-			`replaying,{"bytes_per_second":0.0,"bytes_per_snapshot":149504.0","remote_snapshot_timestamp":1662655501`,
-			nil,
-			"failed to unmarshal",
+			name: "description without local_snapshot_time",
+			//nolint:lll // sample output cannot be split into multiple lines.
+			description: `replaying, {"bytes_per_second":0.0,"bytes_per_snapshot":81920.0,"last_snapshot_bytes":81920,"last_snapshot_sync_seconds":56743,"remote_snapshot_timestamp":1684675261,"replay_state":"idle"}`,
+			info: localstruct{
+				lastsyncduration: durationpb.New(duration),
+				lastsynctime:     nil,
+				lastsyncbytes:    81920,
+			},
+			expectedErr: "",
 		},
 		{
-			"description with no JSON",
-			`replaying`,
-			nil,
-			corerbd.ErrLastSyncTimeNotFound.Error(),
+			name: "description without last_snapshot_sync_seconds",
+			//nolint:lll // sample output cannot be split into multiple lines.
+			description: `replaying, {"bytes_per_second":0.0,"bytes_per_snapshot":81920.0,"last_snapshot_bytes":81920,"local_snapshot_timestamp":1684675261,"remote_snapshot_timestamp":1684675261,"replay_state":"idle"}`,
+			info: localstruct{
+				lastsyncduration: nil,
+				lastsynctime:     timestamppb.New(time.Unix(1684675261, 0)),
+				lastsyncbytes:    81920,
+			},
+			expectedErr: "",
+		},
+		{
+			name: "description with invalid JSON",
+			//nolint:lll // sample output cannot be split into multiple lines.
+			description: `replaying,{"bytes_per_second":0.0,"last_snapshot_bytes":81920","bytes_per_snapshot":149504.0","remote_snapshot_timestamp":1662655501`,
+			info: localstruct{
+				lastsyncduration: nil,
+				lastsynctime:     nil,
+				lastsyncbytes:    0,
+			},
+			expectedErr: "failed to unmarshal",
+		},
+		{
+			name:        "description with no JSON",
+			description: `replaying`,
+			info: localstruct{
+				lastsyncduration: nil,
+				lastsynctime:     nil,
+				lastsyncbytes:    0,
+			},
+			expectedErr: corerbd.ErrLastSyncTimeNotFound.Error(),
 		},
 	}
 	for _, tt := range tests {
 		tt := tt
 		t.Run(tt.name, func(t *testing.T) {
 			t.Parallel()
-			ts, err := getLastSyncTime(tt.description)
+			teststruct, err := getLastSyncInfo(tt.description)
 			if err != nil && !strings.Contains(err.Error(), tt.expectedErr) {
 				// returned error
-				t.Errorf("getLastSyncTime() returned error, expected: %v, got: %v",
+				t.Errorf("getLastSyncInfo() returned error, expected: %v, got: %v",
```

Comment:
Including `tt.name` in the logs helps when there is a failure.

https://github.com/ceph/ceph-csi/actions/runs/5338117669/jobs/9675045736?pr=3894#step:3:1194
https://github.com/ceph/ceph-csi/actions/runs/5338117669/jobs/9675045736?pr=3894#step:3:1262

the order of the tests may not be the same as  the log message, I think?
https://github.com/ceph/ceph-csi/actions/runs/5338117669/jobs/9675045736?pr=3894#step:3:1271